pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -63,15 +69,13 @@ task trim_reads {
 
     input {
         File    reads_qza
-        
-        String  qza_basename = basename(reads_qza, '.qza')
         #Boolean not_default = false
         String  forward_adapter         = "CTGCTGCCTCCCGTAGGAGT"
         String  reverse_adapter         = "AGAGTTTGATCCTGGCTCAG"
         Int    min_length              = 1
         Boolean keep_untrimmed_reads   = false
         Int     memory_mb = 2000
-        Int     cpu = 1
+        Int     cpu = 3
```

Comment:
Let's just make it an even number--I haven't seen most platforms provide 3 CPUs for a VM
```suggestion
        Int     cpu = 4
```